.github/workflows/playground_frontend_test.yml

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Playground Frontend Test
+
+on:
+  push:
+    paths: ['playground/frontend/**']
+    branches: ['**']
+  pull_request:
+    paths: ['playground/frontend/**']
+    branches: ['**']
+  workflow_dispatch:
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  playground_frontend_test:
+    name: Playground Frontend Test
+    runs-on: ubuntu-latest
+
+    env:
+      FLUTTER_VERSION: '3.3.2'
```

Comment:
Is it the single source of truth for the project's Flutter version? Should it be stored here?

Reply:
Unfortunately, this will be the 9th location where we hardcode the Flutter version, and the workflow for ToB will be the 10th one. We will add it to the list in
- https://github.com/apache/beam/issues/23243

once this is merged. I do not have a solution with less redundancy for now.